openage/convert/main.py

```diff
@@ -213,16 +223,29 @@ def main(args, error):
     root["cfg"].mount(get_config_path())
     args.cfg_dir = root["cfg"]
 
+    auxiliary_files_dir = root["cfg"] / "converter" / "games"
+    avail_game_eds, avail_game_exps = create_version_objects(auxiliary_files_dir)
+
+    game_version = None
+    if args.game_edition is not None:
+        game_version = get_game_version_by_id(
+            args.game_edition, args.game_expansion, avail_game_eds, avail_game_exps)
+        if game_version[0] is None:
+            err(f"Game edition {args.game_edition} is not supported.")
+            return 1
+    else:
+        game_version = get_game_version(srcdir, avail_game_eds, avail_game_exps)
```

Comment:
this is now duplicated with the code in line 67. in order to deduplicate, one common implementation (probably a new function) has to check the the argument values, and autodetect/use the arg accordingly.